modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/onboarding/upgrade/UpgradePlusLayoutExperiment.kt

```diff
@@ -0,0 +1,185 @@
+
+                        Box(
+                            modifier = Modifier.padding(bottom = 20.dp),
+                            contentAlignment = Alignment.Center,
+                        ) {
+                            AutoResizeText(
+                                text = stringResource(state.currentFeatureCard.titleRes(source)),
+                                color = Color.White,
+                                maxFontSize = 22.sp,
+                                lineHeight = 30.sp,
+                                fontWeight = FontWeight.W700,
+                                maxLines = 2,
+                                textAlign = TextAlign.Center,
+                                modifier = Modifier
+                                    .padding(horizontal = 24.dp)
+                                    .fillMaxWidth(),
+                            )
+                        }
+
+                        Spacer(Modifier.height(20.dp))
+
+                        FeatureCards( // TODO: it will be replaced for the new one
+                            state = state,
+                            upgradeButton = state.currentUpgradeButton,
+                            onFeatureCardChanged = onFeatureCardChanged,
+                        )
+                    }
+
+                    Spacer(Modifier.weight(1f))
+                }
+            }
+        }
+
+        if (canUpgrade) {
+            SubscribeButton(onClickSubscribe)
+        }
+    }
+}
+
+@Composable
+private fun SubscribeButton(
+    onClickSubscribe: () -> Unit,
+) {
+    Box(
+        contentAlignment = Alignment.BottomCenter,
+    ) {
+        Column {
+            Button(
+                onClick = onClickSubscribe,
+                shape = RoundedCornerShape(12.dp),
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(horizontal = 16.dp, vertical = 24.dp)
+                    .heightIn(min = 56.dp),
+                colors = ButtonDefaults.buttonColors(
+                    backgroundColor = colorResource(UR.color.plus_gold),
+                ),
+            ) {
+                Column(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(horizontal = 34.dp),
+                    horizontalAlignment = Alignment.CenterHorizontally,
+                ) {
+                    AutoResizeText(
+                        text = stringResource(LR.string.get_pocket_casts_plus),
+                        color = Color.Black,
+                        maxFontSize = 18.sp,
+                        lineHeight = 22.sp,
+                        fontWeight = FontWeight.W600,
+                        maxLines = 1,
+                        textAlign = TextAlign.Center,
+                    )
+                }
+            }
+            Spacer(
+                modifier = Modifier
+                    .windowInsetsBottomHeight(WindowInsets.navigationBars),
```

Comment:
It doesn't feel quite right that this component has padding included. It would make it hard to reuse.

Reply:
Done! [897b10f](https://github.com/Automattic/pocket-casts-android/pull/2635/commits/897b10fa55dfedbbb50ea076624c065c07e59996) I was a copy and paste issue from the original paywall